scripts/correlation_test.py

```diff
@@ -106,13 +117,74 @@ def get_random_score_ratings(user_ratings_list, num):
         ans.append(permute_dict)
     return ans
 
+def reverse(edge):
+    start = edge[0]
+    end = edge[1]
+    result = (end, start)
+    return result
+
+def get_random_edges(user_ratings, graph, num_permutations):
+    '''
+    Takes in user ratings for a particular restaurant, the friend graph, and the number
+    randomly generated permutations to create
+    Return a list of user->list of friends dictionaries
+    '''
+
+    #create the original set of edges
+    edges = []
+    for user in user_ratings:
+        start = user[0]         
+        for friend in graph[start]:
+            e = (start, friend)
+            edges.append(e)
+
+    #the list of randomized adjacency lists to return
+    randomized_graphs = []
+
+    for i in range(num_permutations):
+        randomized_edges = list(edges)
+        #pick two random edges from the list and swap their endpoints
+        for i in range(len(edges)):
+            first = random.choice(randomized_edges)
+            second = random.choice(randomized_edges)
+            while second is first:
+                second = random.choice(randomized_edges)
+            new_first = (first[0], second[1])
+            new_second = (second[0], first[1])
+            randomized_edges.remove(first)
+            randomized_edges.remove(second)
+            #randomized_edges.remove(reverse(first))
+            #randomized_edges.remove(reverse(second))
+            randomized_edges.append(new_first)
+            randomized_edges.append(new_second)
+            #randomized_edges.append(reverse(new_first))
+            #randomized_edges.append(reverse(new_second))
```

Comment:
Why did you comment out the randomized_edges reverse? Because don't you have two duplicate edges to resolve now?